### PR TITLE
BP-1061 - Change from TransferWise to Wise

### PR DESCRIPTION
**Pull request by joaoduraes-tw**
## Context

Move from TransferWise to Wise

## Checklist
- [X] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [X] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: eu.html [hunk 1]
```diff
@@ -170,13 +170,13 @@
 <p ><b><span>5.
 Who are we and how to contact us</span></b></p>
 
-<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;TransferWise
+<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;Wise
```
Comment (amin-abutaleb-tw) [inline eu.html]: Should say this

5.1 Our company information. Wise is the trading name of TransferWise Europe NV/SA, a company incorporated under the law of Belgium with company number 0713.629.988 ("Wise", "we", "us", or "our" as applicable).

File: eu.html [hunk 2]
```diff
@@ -170,13 +170,13 @@
...
-<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;TransferWise
+<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;Wise
 Europe NV/SA is a company incorporated under the law of Belgium with company
-number 0713.629.988 (&quot;TransferWise&quot;, &quot;we&quot;, &quot;us&quot;,
+number 0713.629.988 (&quot;Wise&quot;, &quot;we&quot;, &quot;us&quot;,
 or &quot;our&quot; as applicable).</span></p>
 
 <p ><b><span >5.2 Our Belgian Registered office.</span></b><span>&nbsp;Our
-registered office is TransferWise Europe SA/NV, Avenue Louise 54, Room s52,
+registered office is Wise Europe SA/NV, Avenue Louise 54, Room s52,
```
Comment (amin-abutaleb-tw) [inline eu.html]: Should say this

5.2 Our Belgian Registered office. Our registered office is TransferWise Europe SA/NV, Avenue Louise 54, Room s52, 1050 Brussels, Belgium.

**Review by amin-abutaleb-tw — COMMENTED** [review]
Please change as well in the other files

**Comment by joaoduraes-tw** [comment]
@amin-abutaleb-tw, I did the changes for `eu` and `default`, but not sure about the `intesa-hu` one. I think we might need translations?

**Comment by amin-abutaleb-tw** [comment]
Maybe ask in #hun-translations?

**Review by amin-abutaleb-tw — APPROVED** [review]

**Pull request merged**